infra/makemake/buildbot.nix

```diff
@@ -14,7 +14,6 @@ in {
       admins = ["Erethon" "fricklerhandwerk" "Janik-Haag" "lorenzleutgeb"];
       workersFile = secret "workers";
       github = {
-        user = "ngi-nix-bot";
```

Comment:
This change is made following https://github.com/nix-community/buildbot-nix/pull/186.